lib/mix/lib/releases/assembler.ex

```diff
@@ -497,6 +505,12 @@ defmodule Mix.Releases.Assembler do
       do: :ok
   end
 
+  defp generate_cookie do
+    # 255 is max. atom length, see http://erlang.org/doc/efficiency_guide/advanced.html
+    # Cookies, at least when read from disk, need to be from the printable ASCII range
+    (1..255)|> Enum.map(fn _ -> Enum.random(?\s..?~) end) |> List.to_atom
```

Comment:
I think I prefer `:"#{Enum.map(1..64, fn _ -> Enum.random(?!..?~) end)}"` here. A few reasons:

- 64 characters, same as a sha256 hash, any more than that and it's just needlessly excessive, but 64 chars is still easy to read/manage on a single line
- Start the character range at `!` instead of `\s`, because there is no point in having spaces in the generated cookie, and I'm not sure how spaces might mess up parsing of the vm.args file by Erlang.
- Generate the string directly inside an interpolation expression since it does the same thing but is more concise. This one is obviously subjective, but I personally prefer it.